domains/_gh-uwunotislove-o.uwunotislove.is-a.dev.json

```
@@ -0,0 +1,9 @@
+{
+    "owner": {
+        "username": "uwunotislove",
+        "email": "moody761@hotmail.com"
+    },
+    "record": {
+        "TXT": "0964074425"
```

```suggestion
        "TXT": ["0964074425"]
```
Need to update docs ig

---

i updated